modules/ROOT/pages/asm-secret-group-concept.adoc

```diff
@@ -1,34 +1,14 @@
 = Secret Groups
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
 
 A secret group is a unique secrets vault to which all your secrets are bound. Secrets Manager stores your secrets per secret group, per environment, and per business group.
 
 User access and authorization controls are applied at the environment level. A user who has access on a given environment can access all the secret groups that belong to that environment. Only applications deployed to those specific business groups and environments within your organization can access the secrets stored in each secret group.
 
-Secrets Manager provides both high availability and consistency of stored data, as well as concurrent use functionality for secret renewals. This ensures that all clients use a consistent set of secrets and all users edit the same set of secrets at a given time. 
+Secrets Manager provides both high availability and consistency of stored data, as well as concurrent use functionality for secret renewals. This ensures that all clients use a consistent set of secrets and all users edit the same set of secrets at a given time.
```

Comment:
This paragraph should be deleted. Concurrent use and consistency were eliminated with secret group locks.

Reply:
agreed. We should delete this paragraph

Reply:
Removed this paragraph.